crates/router/tests/connectors/nexinets.rs

```diff
@@ -32,22 +33,26 @@ impl utils::Connector for NexinetsTest {
     }
 }
 
-static CONNECTOR: NexinetsTest = NexinetsTest {};
-
-fn get_default_payment_info() -> Option<utils::PaymentInfo> {
-    None
-}
-
-fn payment_method_details() -> Option<types::PaymentsAuthorizeData> {
-    None
+impl NexinetsTest {
+    fn get_payment_data() -> Option<PaymentsAuthorizeData> {
+        Some(PaymentsAuthorizeData {
+            currency: storage_models::enums::Currency::EUR,
+            payment_method_data: types::api::PaymentMethodData::Card(api::Card {
+                card_number: Secret::new(String::from("4012001038443335")),
+                ..utils::CCardType::default().0
+            }),
+            router_return_url: Some("https://google.com".to_string()),
+            ..utils::PaymentAuthorizeType::default().0
+        })
+    }
 }
 
 // Cards Positive Tests
 // Creates a payment using the manual capture flow (Non 3DS).
 #[actix_web::test]
 async fn should_only_authorize_payment() {
     let response = CONNECTOR
-        .authorize_payment(payment_method_details(), get_default_payment_info())
+        .authorize_payment(NexinetsTest::get_payment_data(), None)
```

Comment:
why NexinetsTest::get_payment_data()? payment_method_details() won't work?

Reply:
changed it to `payment_method_details()`